src/ROUTES.js

```diff
@@ -69,7 +69,7 @@ export default {
     REPORT_WITH_ID_DETAILS_SHARE_CODE: 'r/:reportID/details/shareCode',
     getReportShareCodeRoute: (reportID) => `r/${reportID}/details/shareCode`,
     SELECT_YEAR: 'select-year',
-    getYearSelectionRoute: (minYear, maxYear, currYear, backTo) => `select-year?min=${minYear}&max=${maxYear}&year=${currYear}&backTo=${backTo}`,
```

Comment:
@WoLewicki I am not sure about this solution, the `backTo` parameter is used so we can reuse the year selection route in different datepickers in the app, now we only have it in the DOB but we are currently adding it to the manual requests flow or bank account flow. We need to know what page to go back to from the year page.